### DISCUSSION TITLE
Is it possible to pass a seed to JWK.generate() and make the output deterministic?

### DISCUSSION THREAD

**Discussion by gjgd** [panva/jose#87] [Q&A]
I would like to have a deterministic output when using `JWK.generate()` in my tests

**Comment by panva** [comment]
> Is it possible to pass a seed to JWK.generate() and make the output deterministic?

It is not, these methods are wrappers around node's [crypto.generateKeyPair](https://nodejs.org/api/crypto.html#crypto_crypto_generatekeypair_type_options_callback)
